.github/workflows/trigger_staging_deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Trigger release to staging
+
+on:
```

Comment:
For the purpose of self-education and understanding the syntax - which resources would you recommend to decipher this (I'm assuming it's terraform or its variant)

Reply:
No, this is a github action https://docs.github.com/en/actions

Reply:
👍